### DISCUSSION TITLE
Windows cannot be installed to this disk. The selected disk is of the GPT partition style.

### DISCUSSION THREAD

**Discussion by Waldorf3** [ventoy/Ventoy#2586] [Q&A]
I already have Win 11 installed on this disk, but want to have a second Win 11 instance in order to separate business and pleasure.

So I've made an empty partition using the Win 11 partition manager, and boot from wintoy in order to install Win 11 on the empty partition, but I get the error "Windows cannot be installed to this disk. The selected disk is of the GPT partition style.".  This is a new(ish) PC, and already running UEFI, and the disk is obviously GPT, so why do I get this error?

I tried both the avaiable options in ventoy, I can't remember the exact wording, but something like "normal mode" and "wintoy mode", but same error either way.  

When I Google keywords, I get a link to setting up Rufus to boot in UEFI mode. Surely that must also be possible in Ventoy?

What could be going wrong?

**Comment by steve6375** [comment]
Does the ventoy menu system show efi or bios as the boot mode?

**Reply by Waldorf3** [reply]
Do you mean installing or booting ventoy?  

Booting ventoy the only options I find are either "normal mode" or "winboot mode".  

Installinv ventoy I installed GPT mode.

Not sure what you mean by "ventoy menu system"?

**Comment by steve6375** [comment]
When you boot to the Ventoy Manu system, it shows you the ventoy version number and the boot mode
Have you booted in uefi mode?
If so it will show it next to the version number

**Reply by Waldorf3** [reply]
Ah yes, 1.0.95 BIOS.  How do I make it boot in UEFI mode?

**Comment by steve6375** [comment]
Most computers have A boot hotkey like F8 or F11, depends on bios 
Then select usb uefi boot option

**Reply by ourhelpdesk** [reply]
Ah, right. I had 3 entries for ventoy, the last one was for uefi. It kinda thew me off that it mentioned second partition, but it works.

![image](https://github.com/ventoy/Ventoy/assets/97168695/9cac8181-0b95-42b3-8280-975acad03264)

Bonus question: Any idea how I can get rid of those old entries no longer active.  From the bios setup it seems I can only hide them, not delete them. The boot list on Windows has only the windows instance.

**Comment by steve6375** [comment]
No idea, depends on bios. Power off or reset bios settings?
